### PR TITLE
Add FamilySearch JavaScript SDK

### PR DESCRIPTION
**Pull request by justincy**

### CODE REVIEW

File: files/familysearch-javascript-sdk/update.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "familysearch-javascript-sdk",
+  "repo": "FamilySearch/familysearch-javascript-sdk",
+  "files": {
+    "include": ["*.js"]
```
Comment (jimaek) [inline files/familysearch-javascript-sdk/update.json]: This wont work. There is no `familysearch-javascript-sdk.min.js` in here https://github.com/FamilySearch/familysearch-javascript-sdk/tree/v1.0.0-beta

Comment (justincy) [inline files/familysearch-javascript-sdk/update.json]: @jimaek Correct. That release was done incorrectly. In all other tags, such as the latest [1.0.0-beta3](https://github.com/FamilySearch/familysearch-javascript-sdk/tree/1.0.0-beta3) the files do exist.

Comment (jimaek) [inline files/familysearch-javascript-sdk/update.json]: I see, then please replace `*.js` with `familysearch-javascript-sdk.min.js`. No need for the unminified file in production

Comment (justincy) [inline files/familysearch-javascript-sdk/update.json]: @jimaek Done.

**Comment by jimaek** [comment]
@jsdelivrbot squash and merge

**Pull request closed without merging**